contrib/cf-remote/cf_remote/remote.py

```diff
@@ -190,6 +232,7 @@ def install_host(
         extension = None
         if "package_tags" in data and "msi" in data["package_tags"]:
             extension = ".msi"
+            del data["package_tags"]["msi"]
```

Comment:
What does this have to do with CentOS 6?

Reply:
`data["package_tags"]` is now appended (extending) tags below, to get the tags from os-release stuff. As a side effect, it means that the msi tag would be added "twice" both as an extension and as a tag. This seemed unnecessary.